docs/admin/release_notes/version_1.0.md

```diff
@@ -0,0 +1,30 @@
+# v1.0 Release Notes
+
+This document describes all new features and changes in the release `1.0`. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Release Overview
+
+## v1.0.1 (2021-03-09)
+
+### Added
+
+- #3 - Data exports from NetBox v2.10.5 are now permitted for importing.
```

Comment:
All Issue/PR numbers should be either links or inline code, otherwise it's parsed as markdown heading format.

```suggestion
- `#3` - Data exports from NetBox v2.10.5 are now permitted for importing.
```

or 

```suggestion
- [#3](https://github.com/nautobot/nautobot-plugin-netbox-importer/issues/3) - Data exports from NetBox v2.10.5 are now permitted for importing.
```